<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImportCSV_Step1</class>
 <widget class="QDialog" name="DlgImportCSV_Step1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Importer, Step 1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter or paste comma-separated player names below, one per line. The format is:&lt;/p&gt;&lt;p&gt;&lt;code&gt;Last name, given name, m/f, team name, &lt;i&gt;category 1, category 2, ...&lt;/i&gt;&lt;/code&gt;&lt;/p&gt;&lt;p&gt;Non-existing teams will be newly created. &amp;quot;m&amp;quot; or &amp;quot;f&amp;quot; denotes &amp;quot;male&amp;quot; or &amp;quot;female&amp;quot;. Category names are optional; if provided, the player will be added to the categories if possible.&lt;/p&gt;&lt;p&gt;All input is case-sensitive (e.g., &amp;quot;Team 1&amp;quot; is different from &amp;quot;team 1&amp;quot; and is different from &amp;quot;Team1&amp;quot;).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="txtBox">
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Load from file...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Field delimiter</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="rbComma">
            <property name="text">
             <string>Co&amp;mma (,)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbSemi">
            <property name="text">
             <string>Semico&amp;lon (;)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbTab">
            <property name="text">
             <string>Tab</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Additionally, add&lt;br&gt;
all players to:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCat"/>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>Next &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportCSV_Step1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportCSV_Step1</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBox</sender>
   <signal>textChanged()</signal>
   <receiver>DlgImportCSV_Step1</receiver>
   <slot>onContentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onContentChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
